fonts/Lato/OFL.txt

```diff
@@ -1,93 +0,0 @@
-Copyright (c) 2010-2014 by tyPoland Lukasz Dziedzic (team@latofonts.com) with Reserved Font Name "Lato"

```

Comment:
Why were all of these fonts deleted? Weren't they being used?

Reply:
@taoeffect `Lato` was in use but in fewer places. (e.g. only in buttons and some input element)
I realised `Poppin` is the font used for input elements and buttons in both Group Income and the new `groupincome.org` website projects, so thought we would do the same in this project too.